dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/impl/validation/resources/ValidationMessages.nlsprops

```diff
@@ -0,0 +1,89 @@
+pathItemOperationNullParameter=The PathItem object must contain a valid path. The list of parameters from the \"{0}\" operation from the \"{1}\" path contains a null parameter.
+pathItemOperationParameterNotDeclaredSingle=The PathItem object must contain a valid path. The \"{0}\" operation from the \"{1}\" path defines one path parameter that is not declared: \"{2}\".
+pathItemOperationParameterNotDeclaredMultiple=The PathItem object must contain a valid path. The \"{0}\" operation from the \"{1}\" path defines \"{2}\" path parameters that are not declared: \"{3}\".
+pathItemOperationNoPathParameterDeclared=The PathItem object must contain a valid path. The \"{0}\" operation of the \"{1}\" path does not define a path parameter that is declared: \"{2}\".
+pathItemInvalidFormat=The PathItem object must contain a valid path. The format of the \"{0}\" path is invalid.
+
+pathsRequiresSlash=The Paths object must contain a valid path. The \"{0}\" path does not begin with a slash.
+
+referenceExternalOrExtension=The \"{0}\" value is an external reference or an extension. No validation available.
+referenceNotPartOfModel=The specified reference \"{0}\" is not part of the model components.
+referenceNotValid=The specified reference \"{0}\" is not valid.
+referenceNotValidFormat=The \"{0}\" reference is not in a valid format.
+referenceNull=The \"{0}\" model components object is null or $ref \"{1}\" is null.
+referenceToObjectInvalid=The \"{0}\" value is an invalid reference for \"{1}\".
+
+responseMustContainOneCode=The Responses object must contain at least one response code.
+responseShouldContainSuccess=The Responses object should contain at least one response code for a successful operation.
+
+securityRequirementNotDeclared=The \"{0}\" name provided for the SecurityRequirement object does not correspond to a declared security scheme.
+securityRequirementScopeNamesRequired=The SecurityRequirement object''s \"{0}\" field should be a list of scope names required for execution, but is: \"{1}\".
+securityRequirementFieldNotEmpty=The SecurityRequirement object''s \"{0}\" field should be empty, but is: \"{1}\".
+securityRequirementIsEmpty=The SecurityRequirement object must not be empty.
+
+securitySchemeInFieldInvalid=The \"{0}\" SecurityScheme object has an error. The value of its \"in\" field is \"{1}\", but must be one of (\"query\", \"header\", \"cookie\").
+securitySchemeInvalidURL=The SecurityScheme object must contain a valid URL. The \"{0}\" value specified for the URL is not valid.
+securitySchemeNonApplicableField=One or more fields defined are not applicable to a SecurityScheme instance of type \"{0}\".
+
+serverVariableNotDefined=The \"{0}\" variable in the Server object is not defined.
+serverInvalidURL=The Server object must contain a valid URL. The \"{0}\" value specified for the URL is not valid.
+
+requiredFieldMissing=Validation error at location: \"{0}\". Required \"{1}\" field is missing or is set to an invalid value.
```

Comment:
A validation error occurred at the following location: \"{0}\". The required \"{1}\" field is missing or is set to an invalid value.